lib/databricks.js

```diff
@@ -261,7 +267,32 @@ export async function queryStream(req, res, connection) {
 
       stream.on("error", reject);
 
-      stream.pipe(JSONStream.stringify("", "\n", "\n")).pipe(res);
+      stream
+        .pipe(
+          new Transform({
+            objectMode: true,
+            transform(chunk, encoding, cb) {
```

Comment:
Minor preference, but I'd lean towards calling the third argument `callback` rather than `cb`, just for readability.